### PR TITLE
Update UG and modify addqn error message

### LINKED ISSUE #211: [PE-D] Addqn command not working as expected
Entered the following command

Got an error statating lesson not specified. According to UG, no lesson parameted is required

This was the command format given 

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: samlsm/ped#2

### PR DESCRIPTION
**Pull request by chunyongg**
Close #211

- Add memos to UG
- Improve error message for `addqn` when user not in `viewlsn` or `viewgrp`

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -300,6 +300,9 @@ You can use this command to add a new tutorial lesson for a specified tutorial g
 
 `addlsn grp/GROUP_NAME lsn/LESSON_NAME`
 
+> :memo: LESSON_NAME follows the format 'X-Y' where X is the week number, and Y is the lesson number. There can be 
+>more than 9 weeks (as in 10-1) but not more than 9 lessons in a week (1-10 would be invalid).
```
Comment (neoruien) [inline docs/UserGuide.md:304]: There can be 10 lessons in a week (lesson 1-0 to 1-9)
So on a side note, should we allow users to create lesson 1-0?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -811,6 +814,8 @@ in the future.
 
 `addqn qn/QUESTION`
 
+> :memo: You must navigate to a lesson view through `viewlsn` before this command can be used.
```
Comment (neoruien) [inline docs/UserGuide.md]: Perhaps changing to
"You must navigate to the tutorial lesson page through `viewlsn` before this command can be used"
would be clearer

Comment (chunyongg) [reply docs/UserGuide.md]: Thanks, I amended this!

File: docs/UserGuide.md (hunk 1 above)
Comment (chunyongg) [reply docs/UserGuide.md:304]: Good catch! I changed it to allow only lessons 1-1 to 1-9, so it's now 9 lessons.
Added test case for it too.

**Review by neoruien — APPROVED** [review]
LGTM

**Pull request merged**